src/pages/Dashboard/Modals/BorrowRepay/AccountData/index.tsx

```diff
@@ -34,13 +34,14 @@ const AccountData: React.FC<IAccountDataProps> = ({
   const styles = useStyles();
   const { account } = React.useContext(AuthContext);
 
-  const { assets, userTotalBorrowBalanceCents, userTotalBorrowLimitCents } = useUserMarketInfo({
+  // TODO: handle loading state
+  const { data: getUserMarketInfoData } = useGetUserMarketInfo({
```

Comment:
Many of the changes in this PR are checking if the value returned exists. We could also leverage react query using placeholders

Reply:
That's a good point yeah. I originally removed the placeholders to see if that causing issues, but now they can be put back for sure. I'll update the PR